webapp/tests/karma/unit/services/calendar-interval.js

```diff
@@ -15,7 +15,6 @@ describe('service module', () => {
     }
   });
 
-
```

Comment:
This should be converted to a mocha test now, right? Have you left it as is for expediency?

Reply:
I have copied the test in the shared-library, so it has the mocha variant. 
I left the karma one in to prove that the new lib still works :) I can remove it now.